multiuser/permission/che-multiuser-permission-workspace/src/test/java/org/eclipse/che/multiuser/permission/workspace/server/spi/tck/WorkerDaoTest.java

```diff
@@ -195,10 +195,13 @@ public void shouldReturnEmptyListIfWorkersWithSuchUserIdDoesNotFound() throws Ex
   }
 
   /* WorkerDao.removeWorker() tests */
-  @Test
+  @Test(
+      expectedExceptions = NotFoundException.class,
+      expectedExceptionsMessageRegExp = "Worker of workspace 'ws1' with id 'user1' was not found.")
```

Comment:
I think it would be better not to clarify here expected message that should be thrown not from tested method.
Please consider reworking as the following example https://github.com/eclipse/che/blob/40ee503a054b60c7b6dc05928d6c0f8d791b2745/multiuser/integration-tests/che-multiuser-cascade-removal/src/test/java/org/eclipse/che/multiuser/integration/jpa/cascaderemoval/JpaEntitiesCascadeRemovalTest.java#L321